dashboard/dashboard.js

```diff
@@ -0,0 +1,13 @@
+function openTab(evt, cityName) {
+    var i, tabcontent, tablinks;
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+      tabcontent[i].style.display = "none";
+    }
+    tablinks = document.getElementsByClassName("tablinks");
+    for (i = 0; i < tablinks.length; i++) {
+      tablinks[i].className = tablinks[i].className.replace(" active", "");
```

Comment:
use remove and add methods to add or remove class names in element
tablinks[i].classList.remove('active');

evt.currentTarget.classList.add('active');

Since adding class name by + will add redundance to class name list, not a good practice